scripts/cypress/plugins/index.js

```diff
@@ -12,10 +12,69 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+// @ts-check
+const path = require('path');
+const { startDevServer } = require('@cypress/webpack-dev-server');
+const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
+const tsConfigBase = require('../../../tsconfig.base.json');
```

Comment:
you're accessing this twice, I suggest to unify this:

```js

const tsConfigBasePath = path.resolve(__dirname, '../../../tsconfig.base.json')
/**
* @type {import('.....').TsConfig}
*/ 
const tsConfigBase = JSON.parse(tsConfigBasePath)
const tsPaths = new TsconfigPathsPlugin({configFile:tsConfigBasePath})

module.exports = (on, config) => { 
 // do stuff ....
 }
```

with that you can also remove `"resolveJsonModule": true` from  `scripts/tsconfig` which im not confident to add at this moment 😅

Reply:
Good idea, done in [
0ed3afa](https://github.com/microsoft/fluentui/pull/22234/commits/0ed3afa055e0fc623658ae9e5bcf07c9d57b6960)